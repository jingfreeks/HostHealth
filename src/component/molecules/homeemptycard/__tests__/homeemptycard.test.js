import React from 'react';
import {render} from '@testing-library/react-native';
import HomeEmptyCard from '../homeemptycard';

describe('Home Empty Card Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<HomeEmptyCard />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
