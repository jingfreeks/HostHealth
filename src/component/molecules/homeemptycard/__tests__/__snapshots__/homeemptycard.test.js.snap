// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Empty Card Component Should work as expected to get snapshot 1`] = `
<RCTSafeAreaView
  style={
    [
      {},
    ]
  }
>
  <View
    alignType="center"
    style={
      [
        {
          "alignItems": "center",
          "alignSelf": "center",
          "flexDirection": "column",
        },
      ]
    }
  >
    <Image
      resizeMode="contain"
      size={150}
      style={
        [
          {
            "height": 294.2647058823529,
            "width": 294.2647058823529,
          },
        ]
      }
    />
    <Text
      style={
        [
          {
            "color": "#000000",
            "fontFamily": "Avenir",
            "fontSize": 16,
            "fontWeight": "400",
          },
        ]
      }
    />
  </View>
</RCTSafeAreaView>
`;
